Atomic/install.py

```diff
@@ -133,23 +162,51 @@ def install(self):
             be.syscontainers.set_args(self.args)
             return be.install(self.image, self.name)
 
+        installation = None
+        if storage == 'docker' and not args_system:
+            if self.args.system_package == 'build':
+                raise ValueError("'--system-package=build' is not supported for docker backend")
+            labels = img_obj.labels or {}
+            # this means --system-package=yes
+            # we actually don't care about value of the label
+            if 'atomic.has_install_files' in labels:
+                temp_dir = tempfile.mkdtemp()
+                try:
+                    installation = self.build_rpm_for_docker_backend(img_obj.id, temp_dir, labels)
+                    self.syscontainers.install_rpm(installation.destination_path)
```

Comment:
same issue here: the code is Docker specific and should stay in `backends/_docker.py`.

Also here we are using a syscontainers function for Docker containers. the refactor I suggested earlier will help to clean this up as well